File: JS/Easy/461. Hamming Distance.js
/**
 * @param {number} x
 * @param {number} y
 * @return {number}
 */
var hammingDistance = function(x, y) {
    var orx = x ^ y;
    var ans = 0;
    while (orx) {
        orx &= (orx - 1);
        ++ans;
    }
    return ans;
};

File: C#/461. Hamming Distance.cs
public class Solution {
    public int HammingDistance(int x, int y) {
        int orx = x ^ y;
        int ans = 0;
        while (orx != 0) {
            orx &= (orx - 1);
            ++ans;
        }
        return ans;
    }
}